### DISCUSSION TITLE
Primeng/themes Aura Compiled with problems:

### DISCUSSION THREAD

**Discussion by emircandmir** [primefaces/community#3275] [PrimeNG]
![image](https://github.com/user-attachments/assets/8a4bf22f-da5e-406a-b1a9-49f9816dabe9)
I updated my project from primeNg 17 to 19, but I am having trouble with the theme. Previously, @import "primeng/resources/primeng.css" in style.css; We were doing it, but it was removed, now I want to add the Aura theme as mentioned in the doc, but I encounter this error in the compiled section.
### Error: Module not found: Error: Package path ./Aura is not exported from package.

**Comment by qwadrox** [comment]
@emircandmir The import should be:

```diff
+import Aura from '@primeng/themes/aura';
-import Aura from '@primeng/themes/Aura';
```

make sure to install `@primeng/themes` as a dep. 

https://primeng.org/installation#download

**Reply by emircandmir** [reply]
thx for the answer but the error is still the same 
### Error: Module not found: Error: Package path ./aura is not exported from package

here a pack.jsn 
![image](https://github.com/user-attachments/assets/9a0a3002-4a24-44da-b216-b838029070d8)
![image](https://github.com/user-attachments/assets/22eac81e-5385-48b9-83e7-06670e31a40e)

**Reply by qwadrox** [reply]
Hmm. I don't know what it can be. When do you see this error build time? Cause I don't see the import error on the screenshot. Did you try to delete the node_module and package-lock.json and reinstall? Also, check tsconfig.json for path overrides.

**Reply by emircandmir** [reply]
Yes, I deleted and reinstalled it a few times. I think the problem is here **import Aura from '@primeng/themes/aura'**;  The link takes me to index.d.ts, but shouldn't it take me to index.mjs? 
![image](https://github.com/user-attachments/assets/c2e4ef30-50ff-4aa1-8a12-1616e24b536d)

**Reply by qwadrox** [reply]
No that's fine. I think you will have something with the angular.json or the tsconfig.json. Try to compare it to a new app's configuration.
https://stackblitz.com/edit/stackblitz-starters-zq7k5vgq?file=src%2Fmain.ts
I created this from scratch and everything works.  
Can you show your tsconfig?

**Comment by emircandmir** [comment]
Thx, I will make the comparison
![image](https://github.com/user-attachments/assets/0ddd0459-dbb7-4542-80d0-cbe3fb42b429)

**Reply by emircandmir** [reply]
The problem is `module : CommonJS` module: theme needs to be `module : ES2022 `for it to work, but since a library I used in my project did not work in ES2022, I had to make it CommonJS.

**Reply by qwadrox** [reply]
I am glad you found it.
